README.md

```diff
@@ -20,28 +24,69 @@ Or install it yourself as:
 
     $ gem install ssm_config
 
+## Setup
+
+To utilize ActiveRecords, create the following model:
+```
+rails generate model SsmConfigRecord file:string accessor_keys:string value:string
+```
+
+When migrating a file to the ActiveRecord, it is important to correctly input the accessor keys. The field `accessor_keys` represents a hashkey corresponding to a value in the hash: for the sequence of keys used to access a value, the corresponding accessor key will be the keys concatentated with a comma delimiter. For example, if `hash[:key1][:key2][:key3] = value`, the corresponding accessor key would be the string `"key1,key2,key3"`. In the case that there is an array, we include the index embraced by brackets. Consider the following hash:
+
+```yml
+any:
+  build:
+    docker:
+      - image: value1
+    steps:
+      - value2
+      - run: value3
+```
+The accessor keys for `value1`, `value2`, and `value3` would be `"build,docker,[0],image"`, `"build,steps,[0]"`, and `"build,steps,[1],run"`, respectively.
+
+⚠️ **NOTE:** ⚠️ There should be no file name called `cache`, as this will call a method of the `SsmConfig` class. Also, all values read from the table will be strings.
```

Comment:
Maybe we can rename the `cache` method into something a bit more descriptive of what it's doing/storing. Usually in memoization (`||=`) we would name the method the same as the `@` variable. So maybe we just name the method `last_set` or maybe rename the whole thing into `last_processes_times`? Open to suggestions.